assets/src/styles/header.scss

```diff
@@ -0,0 +1,48 @@
+#skip a {
+  position:absolute;
+  left:-10000px;
+  top:auto;
+  width:1px;
+  height:1px;
+  overflow:hidden;
+}
+
+#skip a:focus
+{
+  position:static;
+  width:auto;
+  height:auto;
+}
+.usgs-banner-container {
+  background-color: #000000;
+  a {
+    color: white;
+    text-decoration: none;
+  }
+
+  .usgs-logo {
+    display: inline-block;
+    width: 100px;
+    @media screen and (min-width: $small-screen) {
+      width: 178px;
+    }
+  }
+  .usgs-header-links-container {
+    font-size: $small-font-size;
+    @media screen and (max-width: $small-screen) {
+      #usgs-home-link {
+        font-size: 0px
+      }
+    }
+    @media screen and (min-width: $small-screen) {
```

Comment:
Minor, but looks like we have a mixin we can use here: https://github.com/18F/web-design-standards/blob/develop/src/stylesheets/core/_utilities.scss#L46-L50